schema/packs/strict/items.json

```diff
@@ -318,79 +321,59 @@
-                                        "disabled_img": {
-                                            "$ref": "#/$defs/disabled_img"
-                                        },
-                                        "disabled_img_mods": {
-                                            "$ref": "#/$defs/disabled_img_mods"
-                                        },
-                                        "codes": {
-                                            "$ref": "#/$defs/codes"
-                                        }
-                                    }
+                    "item_right": {
+                        "description": "Code identifier of the right item.",
+                        "type": "string"
+                    },
+                    "images": {
+                        "description": "Array of images and states (up to 4, on/off for each item).",
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "required": [
+                                "left",
+                                "right"
+                            ],
+                            "additionalProperties": false,
+                            "properties": {
+                                "left": {
+                                    "description": "State of left item.",
+                                    "type": "boolean"
+                                },
+                                "right": {
+                                    "description": "State of right item.",
+                                    "type": "boolean"
+                                },
+                                "img": {
+                                    "$ref": "#/$defs/img"
+                                },
+                                "img_mods": {
+                                    "$ref": "#/$defs/img_mods"
+                                },
+                                "disabled_img": {
+                                    "$ref": "#/$defs/disabled_img"
                                 },
-                                "left_item": false,
-                                "right_item": false
+                                "disabled_img_mods": {
+                                    "$ref": "#/$defs/disabled_img_mods"
+                                },
+                                "name": {
+                                    "$ref": "#/$defs/name_stage"
+                                }
                             }
-                        },
-                        "required": [
-                            "images"
-                        ]
+                        }
                     }
+                },
+                "additionalProperties": false,
+                "required": [
+                    "item_left",
+                    "item_right",
+                    "images",
+                    "codes"
```

Comment:
Hm, I may have been a bit trigger happy. Is this correct?

Reply:
Codes can be used for placement in the layout - the `required` is not really necessary, but we keep it for consistency with other item types.